src/process.cpp

```diff
@@ -36,23 +47,35 @@ string Process::Ram() const { return ram; }
 string Process::User() const { return user; }
 
 // TODO: Return the age of this process (in seconds)
-long int Process::UpTime() const { return uptime; }
+long int Process::UpTime() const { return proc_uptime; }
 
 // TODO: Overload the "less than" comparison operator for Process objects
 // REMOVE: [[maybe_unused]] once you define the function
 bool Process::operator<(Process const& a) const {
-  return a.CpuUtilization() > this->CpuUtilization();
-  // return true;
+  return a.CpuUtilization() < this->CpuUtilization();
+}
+
+// In a given string this function looks for a a specific string 
+//                           and replaces it with a given string.
+// This helper function only serves Process::Cpu_Mem_Utime()
+string findAndReplace(string subject, string lookfor, string replace_with) {
+  auto x = subject.find(lookfor);
+  if (x > subject.size()) {
+    return subject;
+  }
+  auto y = subject.replace(x, lookfor.length(), replace_with);
```

Comment:
```suggestion
  subject.replace(x, lookfor.length(), replace_with);
```
declaring y is unnecesary

Reply:
agreed!